content/learn/publication-opportunities.md

```diff
@@ -0,0 +1,126 @@
+---
+Title: Pre- & Post-Publication Opportunities
+weight: 6400
+type: essay
+abstract:  "Funding, conferences, awards, and more for your Quire project"
+---
+
+## PRE-PUBLICATION
```

Comment:
@dimrkh Let's make both this and Post-Publication lower case. It looks really intense in the sidebar menu.